service/src/main/java/bio/terra/workspace/common/utils/IamRoleUtils.java

```diff
@@ -3,35 +3,29 @@
 import bio.terra.workspace.common.exception.EnumNotRecognizedException;
 import bio.terra.workspace.service.iam.model.ControlledResourceIamRole;
 import bio.terra.workspace.service.resource.controlled.AccessScopeType;
-import java.util.Collections;
-import java.util.List;
 
 public final class IamRoleUtils {
 
   private IamRoleUtils() {}
 
   /**
-   * Build the list of IAM roles for this user. If the resource was initially shared, we make the
-   * cloned resource shared as well. If it's private, the user making the clone must be the resource
-   * user and becomes EDITOR, READER, and WRITER on the new resource.
+   * Determine the IAM role for this user. If the resource was initially shared, we make the cloned
+   * resource shared as well. If it's private, the user making the clone must be the resource user
+   * and becomes EDITOR.
    *
    * @param accessScope - private vs shared access
-   * @return list of IAM roles for the user on the resource
+   * @return IAM role for the user on the resource
    */
-  public static List<ControlledResourceIamRole> getIamRolesForAccessScope(
-      AccessScopeType accessScope) {
+  public static ControlledResourceIamRole getIamRolesForAccessScope(AccessScopeType accessScope) {
```

Comment:
Is this method specific to cloning resources? I wonder if the method name could make that more clear: `getCloneDestinationIamRolesForAccessScope` or something.

Reply:
This is an odd place to have this code. I'm going to make a TODO and a ticket rather than further expand this PR.

There is a larger issue here. I did not address cloning of application-owned resources. Or preventing them from being cloned. We don't have a design for cloning application resources thought through. So another ticket for that.

Reply:
OK sounds good. I will say for Leonardo-owned resources, we don't expect anything to be cloned.